packages/mobile/src/app/actions.ts

```diff
@@ -99,6 +101,16 @@ interface MinAppVersionDeterminedAction {
   minVersion: string | null
 }
 
+interface PontoFeatureFlagSetAction {
+  type: Actions.SET_PONTO_FEATURE_FLAG
+  enabled: boolean
+}
+
+interface KotaniFeatureFlagSetAction {
+  type: Actions.SET_KOTANI_FEATURE_FLAG
+  enabled: boolean
+}
+
```

Comment:
Fine for now given the short timeline we have, but I think this could be more generic and handle all remote feature flags in a single "REMOTE_FEATURE_FLAGS_UPDATED" action.